lib/remote/createobjecthandler.cpp

```diff
@@ -116,7 +116,7 @@ bool CreateObjectHandler::HandleRequest(
 		return true;
 	}
 
-	if (!ConfigObjectUtility::CreateObject(type, name, config, errors, diagnosticInformation)) {
+	if (!ignoreOnError && !ConfigObjectUtility::CreateObject(type, name, config, errors, diagnosticInformation)) {
```

Comment:
Swap the expressions around the `&&` and explain in your own words why it matters.

Reply:
Ah 😯, well sure, because then it actually wouldn't try to create the object at all if the ``ignore_on_error`` flag is set to true.

Reply:
Exactly.